interface/super/edit_list.php

```diff
@@ -818,147 +810,162 @@ function mysubmit() {
 </head>
 
 <body class="body_top">
+<nav class="navbar navbar-default navbar-fixed-top">
+<div class="container-fluid">
```

Comment:
Just from a learning perspective. When do you choose container vs container-fluid?
container-fluid does seem nice since don't need to set up responsive behavior by rigid screen sizes, but I am guessing there is a drawback?